extensions/tw-colorsearch/src/index.tsx

```diff
@@ -328,10 +353,11 @@ export default function Command() {
           accessories={[{ icon: Icon.Text, text: item.hex.toUpperCase() }]}
           actions={
             <ActionPanel>
-              <Action.CopyToClipboard title="Copy HEX" content={item.hex.toUpperCase()} />
-              <Action.CopyToClipboard title="Copy RGB" content={item.rgb} />
+              <Action.CopyToClipboard title="Copy Hex" content={item.hex.toUpperCase()} />
+              <Action.CopyToClipboard title="Copy Rgb" content={item.rgb} />
+              <Action.CopyToClipboard title="Copy Oklch" content={item.oklch} />
               <Action.CopyToClipboard
-                title="Copy TailwindCSS Name"
+                title="Copy Tailwind Css Name"
```

Comment:
`Tailwind CSS` is a [trademark](https://tailwindcss.com/brand). Let's use the correct spelling.

```suggestion
                title="Copy Tailwind CSS Name"
```